trl/trainer/utils.py

```diff
@@ -452,6 +452,7 @@ def __iter__(self):
                 yield {
                     "input_ids": torch.LongTensor(example),
                     "labels": torch.LongTensor(example),
+                    "attention_mask": [1 for _ in example],
```

Comment:
I think the following would be better, since it will also create a torch tensor:

```suggestion
                    "attention_mask": torch.ones_like(example),
```